pkg/webhook/configuration.go

```diff
@@ -460,7 +461,7 @@ func preUpgrade(openebsNamespace string) error {
 	}
 
 	for _, config := range webhookConfigList.Items {
-		if len(config.Labels["openebs.io/version"]) == 0 {
+		if config.Labels["openebs.io/version"] != version.Current() {
```

Comment:
Instead of deleting for every upgrade can't we handle with patch/update call?
cc: @prateekpandey14

Reply:
Yes, better to patch/update instead of delete based on version